### DISCUSSION TITLE
Any plans on supporting RediSearch 2.8?

### DISCUSSION THREAD

**Discussion by blablacio** [dahomey-technologies/rustis#70] [General]
First of all, thanks for the great library! Using it in dozen of production projects already and it shines!

I was working on a search feature last week for one of my projects and using the latest version of Redis Stack, which uses RediSearch 2.8, and it turns out rustis is not compatible. Seems the response format is different than the one in 2.6.

It's not a big deal for now, but I think in the near future it would be advantageous to migrate to 2.8 as it has some nice features lacking in 2.6.

**Comment by mcatanzariti** [comment]
Hello, 

from release [0.14.0](https://github.com/dahomey-technologies/rustis/releases/tag/0.14.0), all commands in Redis 8.0 are supported, including the last RedisSearch version.
